ctapipe/tools/display_dl1.py

```diff
@@ -137,41 +142,31 @@ class DisplayDL1Calib(Tool):
     telescope = Int(
         None,
         allow_none=True,
-        help='Telescope to view. Set to None to display all '
-             'telescopes.'
+        help="Telescope to view. Set to None to display all " "telescopes.",
     ).tag(config=True)
 
-    extractor_product = tool_utils.enum_trait(
-        ImageExtractor,
-        default='NeighborPeakWindowSum'
+    extractor_product = traits.enum_trait(
+        ImageExtractor, default="NeighborPeakWindowSum"
     )
 
     aliases = Dict(
         dict(
-            max_events='EventSource.max_events',
-            extractor='DisplayDL1Calib.extractor_product',
-            T='DisplayDL1Calib.telescope',
-            O='ImagePlotter.output_path'
+            max_events="EventSource.max_events",
+            extractor="DisplayDL1Calib.extractor_product",
+            T="DisplayDL1Calib.telescope",
+            O="ImagePlotter.output_path",
         )
     )
     flags = Dict(
         dict(
             D=(
-                {
-                    'ImagePlotter': {
-                        'display': True
-                    }
-                },
-                "Display the photoelectron images on-screen as they "
-                "are produced."
+                {"ImagePlotter": {"display": True}},
+                "Display the photoelectron images on-screen as they " "are produced.",
```

Comment:
Extra ""

Reply:
thanks- seems to be due to reformatting (strings like that get concatenated, but there's no need in this case)